### ISSUE TITLE
Move the PNI "about" pages into the wagtailpages/templates/buyersguide dir

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#6477]
Right now the PNI templates are located in `wagtailpages/templates/buyersguide` but the PNI about pages live in wagtailpages/templates/about` - these should live in wagtailpages/templates/buyersguide/about`, instead.

- [ ] move these pages
- [ ] update the code that links to these templates
- [ ] notify @TheoChevalier when this work gets done, so he can make sure those path changes are understood on the l10n side.

**Comment by TheoChevalier** [comment]
@Pomax  Turns out it’s currently busted https://foundation.mozilla.org/fr/privacynotincluded/about/, I got fooled locally because I still had the old `locale` folders around, so it was using translations from these.

I removed the old folders, and now I can reproduce the issue locally. Then I had no clue why it was not using the files in `wagtailpages/templates/about/locale`.
`makemessages` is correctly extracting strings into them, `compilemessages` is processing them, but then for some reason Django don’t pick them at runtime.

Just found out the reason: the locale path is hardcoded here https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/settings.py#L433
We need `'networkapi/wagtailpages/templates/about/locale'`
